tests/support/test_server/lib/test_server/channel.ex

```diff
@@ -14,6 +14,14 @@ defmodule TestServer.Channel do
     {:ok, assign(socket, :payload, payload)}
   end
 
+  def handle_in("disconnect", _payload, socket) do
+    IO.inspect(socket)
+    :sys.get_status(socket.transport_pid) |> IO.inspect(label: "Transport PID status")
```

Comment:
Did you mean to leave these `IO.inspect` calls in?

Reply:
No, that was leftover from before Chris McCord confirmed this was the proper way to kill the connection to simulate web socket errors.  Removed.